src/wowman/nfo.clj

```diff
@@ -25,15 +25,26 @@
   (let [sub-dirs (->> path fs/list-dir (filter fs/directory?) (map fs/base-name) (mapv str))]
     (not (nil? (some ignorable-dir-set sub-dirs)))))
 
-(defn-spec derive ::sp/nfo
+(defn-spec derive ::sp/nfo-v2
   "extract fields from the addon data that will be written to the nfo file"
-  [addon ::sp/addon-or-toc-addon, primary? boolean?]
+  [addon ::sp/nfo-input-minimum, primary? boolean?, game-track ::sp/game-track]
   {;; important! as an addon is updated or installed, the `:installed-version` from the .toc file is overridden by the `:version` online
    ;; later, when comparing installed addons against the catalogue, the comparisons will be more consistent
    :installed-version (:version addon)
-   :name (:name addon) ;; normalised name. once used to match to online addon, we now use source+source-id
-   :group-id (:uri addon) ;; groups all of an addon's directories together. this is a verbose but natural way of specifying source+source-id
-   :primary? primary? ;; if addon is one of multiple addons, is this addon considered the primary one?
+
+   ;; knowing the game track at time of installation lets us include that in any exports and later re-import the correct version 
+   :installed-game-track game-track
+
+   ;; normalised name. once used to match to online addon, we now use source+source-id
+   :name (:name addon)
+
+   ;; groups all of an addon's directories together. it's a little verbose but existed before we were capturing source+source-id
```

Comment:
break on 'source' or shorten

Reply:
removed. source+source-id can't be used because occasionally addons bundle other addons that can live independently with their own source-ids